airflow/providers/microsoft/mssql/hooks/mssql.py

```diff
@@ -70,3 +71,9 @@ def set_autocommit(
 
     def get_autocommit(self, conn: pymssql.connect):  # pylint: disable=c-extension-no-member
         return conn.autocommit_state
+
+
+class MsSqlHookWithOdbcHook(OdbcHook):
+    """Interact with Microsoft SQL Server via ODBC."""
+
+    conn_type = 'mssql'
```

Comment:
Don't we need 


```suggestion
    conn_type = 'mssql'
    conn_name_attr = 'mssql_conn_id'
    default_conn_name = 'mssql_default'
```

Reply:
I think just mssql_default is needed. Since it derives from the OdbcHook, it also gets those class-level variables. But indeed if we want to keep consistency default_conn_name should be mssql_default. the conn_name_attr should be still `odbc_conn_id` not `mssql_conn_id` because that's the name of the parameter initialized in the OdbcHoook.

But yeah - adding it here as well explicitly rather than relying the derived value is a good idea.